app/routes/task_routes.py

```diff
@@ -1 +1,99 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, jsonify
+from ..db import db
+from app.models.task import Task
+from .route_utilities import validate_model, create_model
+from datetime import datetime
+import requests, os
+
+
+bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+
+@bp.post("")
+def create_task():
+    request_body = request.get_json()
+    task_data, status_code = create_model(Task, request_body)
+    return jsonify({"task": task_data}), status_code
+
+
+@bp.get("")
+def get_all_tasks():
+    query = db.select(Task)
+
+    title_param = request.args.get("title")
+    if title_param:
+        query = query.where(Task.title.ilike(f"%{title_param}%"))
+
+    description_param = request.args.get("description")
+    if description_param:
+        query = query.where(Task.description.ilike(f"%{description_param}%"))
+
+    sort_param = request.args.get("sort")
+    if sort_param == "desc":
+        query = query.order_by(Task.title.desc())
+    else:
+        query = query.order_by(Task.title.asc())
+
+    tasks = db.session.scalars(query)
+
+    return [task.to_dict() for task in tasks]
+
+
+@bp.get("/<id>")
+def get_one_task(id):
+    task = validate_model(Task, id)
+    return jsonify({"task": task.to_dict()}), 200
```

Comment:
Recall that the default status code that Flask sends back is 200 so we don't need to explicitly return it here. 

```suggestion
    return {"task": task.to_dict()}
```

Notice on line 39 above that you don't return any status code. Flask will return status code 200 by default there as well.